Lib/importlib/abc.py

```diff
@@ -61,7 +64,13 @@ def find_module(self, fullname, path):
         exists then backwards-compatible functionality is provided for this
         method.
 
+        Deprecated since Python 3.4 in favor of find_spec (available since 3.4)
```

Comment:
The deprecation is mentioned in the above paragraph, so if you would like to add the version info just toss in up there.